### ISSUE TITLE
添加策略组时返回404

### ISSUE THREAD

**Issue by ZhangYet** [CNSRE/ABTestingGateway#36]
您好。
我按照示例，添加策略组的时候：
```
curl 127.0.0.1:8080/ab_admin?action=policygroup_set -d '{"1":{"divtype":"uidsuffix","divdata":[{"suffix":"1","upstream":"beta1"},{"suffix":"3","upstream":"beta2"},{"suffix":"5","upstream":"beta1"},{"suffix":"0","upstream":"beta3"}]},"2":{"divtype":"arg_city","divdata":[{"city":"BJ","upstream":"beta1"},{"city":"SH","upstream":"beta2"},{"city":"XA","upstream":"beta1"},{"city":"HZ","upstream":"beta3"}]},"3":{"divtype":"iprange","divdata":[{"range":{"start":1111,"end":2222},"upstream":"beta1"},{"range":{"start":3333,"end":4444},"upstream":"beta2"},{"range":{"start":7777,"end":2130706433},"upstream":"beta2"}]}}'
```
返回了404错误：
```
<html>
<head><title>404 Not Found</title></head>
<body bgcolor="white">
<center><h1>404 Not Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
```
是否接口有了改动而文档没有更新？

**Comment by BG2BKK** [comment]
接口是没问题的，如果使用了最新的代码，应该是没问题的

On Tue, Jul 5, 2016 at 5:17 PM, Zhang Yet <notifications@github.com> wrote:

> 您好。
> 我按照示例，添加策略组的时候：
>
> curl 127.0.0.1:8080/ab_admin?action=policygroup_set -d '{"1":{"divtype":"uidsuffix","divdata":[{"suffix":"1","upstream":"beta1"},{"suffix":"3","upstream":"beta2"},{"suffix":"5","upstream":"beta1"},{"suffix":"0","upstream":"beta3"}]},"2":{"divtype":"arg_city","divdata":[{"city":"BJ","upstream":"beta1"},{"city":"SH","upstream":"beta2"},{"city":"XA","upstream":"beta1"},{"city":"HZ","upstream":"beta3"}]},"3":{"divtype":"iprange","divdata":[{"range":{"start":1111,"end":2222},"upstream":"beta1"},{"range":{"start":3333,"end":4444},"upstream":"beta2"},{"range":{"start":7777,"end":2130706433},"upstream":"beta2"}]}}'
>
> 返回了404错误：
>
> <html>
> <head><title>404 Not Found</title></head>
> <body bgcolor="white">
> <center><h1>404 Not Found</h1></center>
> <hr><center>nginx</center>
> </body>
> </html>
>
> 是否接口有了改动而文档没有更新？
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CNSRE/ABTestingGateway/issues/36>, or mute the thread
> <https://github.com/notifications/unsubscribe/AA0sgZCNOeeK_4eEvUOZyCI0xmO_cmIfks5qSiExgaJpZM4JE7GO>
> .
>



-- 
黄振栋
bg2bkk@gmail.com
Phone：15010359288
personal site: https://bg2bkk.github.io

**Comment by ZhangYet** [comment]
发现是权限问题，参考[issue 28](https://github.com/CNSRE/ABTestingGateway/issues/28)，即可解决。
建议增加一个FAQ文档。

**Comment by BG2BKK** [comment]
好的
On Tue, Jul 5, 2016 at 6:10 PM, Zhang Yet <notifications@github.com> wrote:

> Closed #36 <https://github.com/CNSRE/ABTestingGateway/issues/36>.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/CNSRE/ABTestingGateway/issues/36#event-712797805>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AA0sgb66ZeZbhJ3JMcUny_4B4St2t1pGks5qSi2PgaJpZM4JE7GO>
> .
>



-- 
黄振栋
bg2bkk@gmail.com
Phone：15010359288
personal site: https://bg2bkk.github.io
